src/Goerli-DssSpell.t.sol

```diff
@@ -18,15 +18,61 @@ pragma solidity 0.6.12;
 
 import "./Goerli-DssSpell.t.base.sol";
 
-interface DenyProxyLike {
-    function denyProxy(address) external;
+interface RwaLiquidationLike {
+    function ilks(bytes32) external returns (string memory, address, uint48 toc, uint48 tau);
+    function bump(bytes32 ilk, uint256 val) external;
+    function tell(bytes32) external;
+    function cure(bytes32) external;
+    function cull(bytes32, address) external;
+    function good(bytes32) external view returns (bool);
 }
 
-interface StarknetLike {
-    function ceiling() external returns (uint256);
+interface RwaUrnLike {
+    function can(address) external view returns (uint256);
+    function lock(uint256) external;
+    function draw(uint256) external;
+    function wipe(uint256) external;
+    function free(uint256) external;
+}
+
+interface RwaOutputConduitLike {
+    function pick(address) external;
+    function push() external;
+}
+
+interface RwaInputConduitLike {
+    function push() external;
 }
 
 contract DssSpellTest is GoerliDssSpellTestBase {
+    RwaLiquidationLike oracle = RwaLiquidationLike(addr.addr("MIP21_LIQUIDATION_ORACLE"));
+
+    // -- RWA008 --
+    bytes32 constant     ilk_008             = "RWA008-A";
+
+    address              rwaOperator_008     = 0x3F761335890721752476d4F210A7ad9BEf66fb45;
+    DSTokenAbstract      rwagem_008          = DSTokenAbstract(addr.addr("RWA008"));
+    GemJoinAbstract      rwajoin_008         = GemJoinAbstract(addr.addr("MCD_JOIN_RWA008_A"));
+    RwaUrnLike           rwaurn_008          = RwaUrnLike(addr.addr("RWA008_A_URN"));
+    RwaInputConduitLike  rwaconduitin_008    = RwaInputConduitLike(addr.addr("RWA008_A_INPUT_CONDUIT"));
+    RwaOutputConduitLike rwaconduitout_008   = RwaOutputConduitLike(addr.addr("RWA008_A_OUTPUT_CONDUIT"));
+
+    BumpSpellRwa008      bumpSpell_008;
+    TellSpellRwa008      tellSpell_008;
+    CureSpellRwa008      cureSpell_008;
+    CullSpellRwa008      cullSpell_008;
+    EndSpellRwa008       endSpell_008;
+
+    // -- RWA009 --
+    bytes32 constant     ilk_009             = "RWA009-A";
+    DSTokenAbstract      rwagem_009          = DSTokenAbstract(addr.addr("RWA009"));
+    GemJoinAbstract      rwajoin_009         = GemJoinAbstract(addr.addr("MCD_JOIN_RWA009_A"));
+    RwaUrnLike           rwaurn_009          = RwaUrnLike(addr.addr("RWA009_A_URN"));
+    address              RWA009_CES_MULTISIG = addr.addr("RWA009_A_OUTPUT_CONDUIT");
+
+    BumpSpellRwa009      bumpSpell_009;
+    TellSpellRwa009      tellSpell_009;
+    CullSpellRwa009      cullSpell_009;
```

Comment:
I'd suggest move all this right above the first RWA specific test "testSpellIsCast_RWA00X_INTEGRATION_BUMP".
So for next week we do not forget to remove parts of it.

Reply:
I wouldn't say for this spell but from the next one I think we should have a specific RWA test file.

Reply:
Sure.